src/backend/distributed/shared_library_init.c

```diff
@@ -358,14 +357,6 @@ _PG_init(void)
 }
 
 
-/* shared library deconstruction function */
-void
-_PG_fini(void)
-{
-	columnar_fini();
-}
```

Comment:
why do we need that ?

Reply:
According to Jeff, we are never calling _PG_fini. It was only created for more symmetry with PG_init and it is causing an issue for when we are trying to build Columnar separately. This is also done on Yanwen's draft PR and I am doing it here for her instead so her changes will be cleaner. I know we haven't made the decision on if we are building columnar separately or not but since this code is never called, I think it would be good to remove anyways for better readability.